app/exposureNotificationsNativeModule.ts

```diff
@@ -19,55 +19,55 @@ export const fetchDiagnosisKeys = async (
 };
 
 export const detectExposuresNow = async (
-  cb: (error: Error) => void,
+  cb: (errorString: string) => void,
```

Comment:
```suggestion
export type ENModuleError = string | null

export const detectExposuresNow = async (
  cb: (error: ENModuleError) => void,
```

my understanding is that this can be either a string or null, so we should probably capture that in the type system.  Additionally its best practice to only use the primitives if the actual intent is that primitive and provide a type alias if the variable represents a useful domain concept.  for one, it is more readable to those without context, and for two it is much easier to update if the ENModuleError ever needs to change in the future.